pymysql/connections.py

```diff
@@ -1117,6 +1117,9 @@ def get_server_info(self):
 class MySQLResult(object):
 
     def __init__(self, connection):
+        """
+        @type connection: Connection
```

Comment:
I prefer `:type connection: Connection` syntax.
https://www.jetbrains.com/pycharm/help/type-hinting-in-pycharm.html#d290117e152